diag/exec/controller/exec_controller.cpp

```diff
@@ -23,22 +24,22 @@ void ExecController::Init(uint16_t service_id) {
     auto sock_ = com::soc::IpcSocket();
     auto hdr = ExecHeader(service_id, 1, (this->flags_.to_ulong() << 3) | this->status_);
     while (!stop_token.stop_requested()) {
-        this->mtx_status_.lock();
-        hdr.SetFlags((this->flags_.to_ulong() << 3) | this->status_);
+        {
+            std::unique_lock<std::mutex> lock(this->mtx_status_);
+            hdr.SetFlags((this->flags_.to_ulong() << 3) | this->status_);
+        }
         sock_.Transmit("SIMBA.EXE", 0, hdr.GetBuffor());
-        AppLogger::Debug("id: "+ std::to_string(hdr.GetServiceID())
+        AppLogger::Warning("id: "+ std::to_string(hdr.GetServiceID())
```

Comment:
Czy to jest potrzebne  bo to sypie tym logiem co sekunde i czy to jest krytyczne ?

Napewno nie Warning jak już Info

Reply:
zapomniałem wywalić, do debugu zmieniłem